File: services/service-node/functions/api/lambda_function.js
exports.handler = async (event) => {
    let query = event.queryStringParameters;
    console.log(query);
    let body = {source: "node", query: query};
    let response = {
        isBase64Encoded: false,
        statusCode: 200,
        headers: {
            'Content-Type': 'application/json'
        },
        body: JSON.stringify(body)
    };
    return response;
};

File: services/service-python/functions/api/lambda_function.py
import json


def handler(e, c):
    query = e.get('queryStringParameters')
    body = {"source": "python", "query": query}
    return {
        'isBase64Encoded': False,
        'statusCode': 200,
        'body': json.dumps(body),
        'headers': {
            'Content-Type': 'application/json'
        }
    }
